### PR TITLE
Docs 2459

### PR DESCRIPTION
**Pull request by Kris-MultiSafepay**
Can be merged when approved ✅

Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/accounts/charges.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+A description for the transaction. The description is displayed in the affiliated account and your account. **Optional**. 
+
+**var1** | string
+
+A variable used to store additional data. **Optional**.
+
+**var2** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/charges.md:86]:
```suggestion
A second variable for storing additional data. **Optional**.
```

File: content/api/accounts/charges.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Your unique identifier for the charge. **Required**. 
+
+**description** | string
+
+A description for the transaction. The description is displayed in the affiliated account and your account. **Optional**. 
+
+**var1** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/charges.md:82]:
```suggestion
A variable for storing additional data. **Optional**.
```

File: content/api/accounts/charges.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+**var2** | string
+
+A variable used to store additional data. **Optional**.
+
+**var3** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/charges.md:90]:
```suggestion
A third variable for storing additional data. **Optional**.
```

File: content/api/accounts/balances.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    },
+    "success": true
+}
+```
+
+{{< /code-block >}}
+{{< description >}}
+## Balances
+
+As a partner or primary account holder, request the balance of an affiliated account.
```
Comment (FrancescaNCook) [inline content/api/accounts/balances.md:36]:
```suggestion
As a partner or primary account holder, use /balances to request the balance of an affiliated account. 
```

File: content/api/accounts/charges.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    },
+    "success": true
+}
+```
+
+{{< /code-block >}}
+{{< description >}}
+## Charges
+
+As a partner or primary account holder, use charges to move funds from an affiliated account's balance to your own.
```
Comment (FrancescaNCook) [inline content/api/accounts/charges.md:53]:
```suggestion
As a partner or primary account holder, use /charges to move funds from an affiliated account balance to your own. 
```

File: content/api/accounts/payouts.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    },
+    "success": true
+}
+```
+
+{{< /code-block >}}
+{{< description >}}
+## Payouts
+
+As a partner or primary account holder, use payouts to move funds from an affiliated account's balance to a connected bank account.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:66]:
```suggestion
As a partner or primary account holder, use /payouts to move funds from an affiliated account balance to a connected bank account. 
```

File: content/api/accounts/payouts.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+}
+```
+
+{{< /code-block >}}
+{{< description >}}
+## Payouts
+
+As a partner or primary account holder, use payouts to move funds from an affiliated account's balance to a connected bank account. 
+
+By default, payouts for affiliated accounts are disabled. Contact your account manager to enable payouts for your affiliated accounts.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:68]:
```suggestion
By default, payouts for affiliated accounts are disabled. To enable payouts for your affiliated accounts, email your account manager at <sales@multisafepay.com>
```

File: content/api/accounts/payouts.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+### Request parameters
+
+**affiliated_id** | query parameter  
+
+The account ID of the affiliated account that is to be paid out. **Required**.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:74]:
```suggestion
The account ID of the affiliated account that you want to move funds from. **Required**.
```

File: content/api/accounts/payouts.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+Your unique identifier for the payout. **Required**. 
+
+**description** | string
+
+A description for the transaction. The description is displayed in the affiliated account. **Optional**. 
+
+**var1** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:94]:
```suggestion
A variable for storing additional data. **Optional**.
```

File: content/api/accounts/payouts.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+**var2** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:98]:
```suggestion
A second variable for storing additional data. **Optional**.
```

File: content/api/accounts/payouts.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+**var3** | string
+
+A variable used to store additional data. **Optional**.
```
Comment (FrancescaNCook) [inline content/api/accounts/payouts.md:102]:
```suggestion
A third variable for storing additional data. **Optional**.
```

File: content/api/payment_methods/post_order_visa.md [hunk 12]
```diff
@@ -116,4 +116,4 @@ Options: true, false. Set to true if you want to display the MultiSafepay paymen
 
 __Note: The ip_address and e-mail address parameters are not required, although their use is recommended to help detect fraudulent payments.__
 
-Read more about [Visa](/payment-methods/credit-and-debit-cards/visa) on our documentation page.
\ No newline at end of file
+Read more about [Visa](/payment-methods/credit-and-debit-cards/visa) on our documentation page.
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post_order_visa.md:119]:
```suggestion
For more information, see [Visa](/payment-methods/credit-and-debit-cards/visa).
```

**Review by FrancescaNCook — DISMISSED** [review]
A few tweaks - see what you think :)

File: content/api/payment_methods/post_order_visa.md [hunk 13]
```diff
@@ -116,4 +116,4 @@ Options: true, false. Set to true if you want to display the MultiSafepay paymen
 
 __Note: The ip_address and e-mail address parameters are not required, although their use is recommended to help detect fraudulent payments.__
```
Comment (FrancescaNCook) [inline content/api/payment_methods/post_order_visa.md:117]:
```suggestion
**Note:** The `ip_address` and `e-mail address` parameters are not required, but we recommend using them to help detect fraudulent payments.
```

File: content/api/accounts/charges.md (hunk 5 above)
Comment (Kris-MultiSafepay) [inline content/api/accounts/charges.md:53]:
```suggestion
As a partner or primary account holder, move funds from an affiliated account's balance to your own. 
```

File: content/api/accounts/payouts.md (hunk 6 above)
Comment (Kris-MultiSafepay) [inline content/api/accounts/payouts.md:66]:
```suggestion
As a partner or primary account holder, move funds from an affiliated account's balance to a connected bank account. 
```

**Pull request merged**